ranger/data/scope.sh

```diff
@@ -94,6 +94,18 @@ handle_extension() {
             jq --color-output . "${FILE_PATH}" && exit 5
             python -m json.tool -- "${FILE_PATH}" && exit 5
             ;;
+	## 3D models
+	stl|off|dxf)
+	    [[ "${PV_IMAGE_ENABLED}" == 'True' ]] && \
+            openscad --colorscheme="Tomorrow Night" --imgsize="1000,1000" -o "/tmp/stl.png" <(echo "import(\"${FILE_PATH}\");"); mv "/tmp/stl.png" "${IMAGE_CACHE_PATH}" && \
```

Comment:
Hmm, using `stl.png` for any and all previews will wreck the cache. Instead, base the name for the cached image on the path to the file being previewed.

Reply:
@toonn fixed, I now use `"/tmp/$(basename "${FILE_PATH}").png"`